web/lib/hooks/useSanityLoader.ts

```diff
@@ -12,6 +23,7 @@ export const useSanityLoader = (image: ImageWithAlt, maxWidth: number, aspectRat
         ? Math.round(width * aspectRatio)
         : Math.round(width * (cropped.height / cropped.width))
 
-      return imageUrlBuilder.width(width).height(height).auto('format').quality(70)
+      return imageUrlBuilder.width(width).height(height).auto('format').quality(70).dpr(devicePixelRatio)
```

Comment:
This doesn't make any difference. I would not recommend adding this since it is more js code for not a reasonable improvement.
I checked it and the issue is with max-width and sizes:

![image](https://github.com/equinor/energyvision/assets/23337266/ed6a8d94-faa4-44ee-b2e6-b70197da5b3c) vs 
![image](https://github.com/equinor/energyvision/assets/23337266/d64b1605-f1cb-48a1-aedd-f22600393367)